docs/Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'jekyll-include-cache'
 group :graphqlDocs_plugins do
  gem "rake"
  gem "graphql"
- gem "rails"
+ gem "rails", ">= 7.1.0"
```

Comment:
Should we pin to a specific version instead of >=

Reply:
yah we can "=" if we want exact or specific version

Reply:
It's better to pin to a version,

Reply:
i just fixed it , but let me know about  tilde (~>) constraint.  This allows your project to use any version of Rails within the 7.1.x series, as long as it’s at least 7.1.0. ,  i see many projects use that

Reply:
Yes, auto bump in minor version is ok.
Thanks.

Reply:
Should there be tests to check rails version is compatible with project libs?

Reply:
“Check the changes in this PR to ensure they won't cause issues with your project.”